data/src/error_analysis/erroranalysis.py

```diff
@@ -0,0 +1,70 @@
+"""Error Analysis tool for G2P.
+
+This script assumes 2 input files. a) Covering Grammer b) Test output. 
+CG file  : Each line contain graphime and their corresponding pronunciation seperated by a tab
+Test File: contains three attributes in each line seperated by a tab. The attributes are - orthography, expected pronunciation, and hypothesized pronunciation."""
```

Comment:
"file" not "File". The use of - is weird here, I'd just give an example line instead?

Reply:
I wanted to highlight the ordering of the column in the above sentence. I am waiting for another team member to send me a test file related to WikiPron. I will remove the '-' once I have a test file.

Reply:
One possibility would be to give an example of the format, with or without the explanation of what the columns mean. For instance it could be something like:

```
espresso ɛ s p ɹ ɛ s ə ʊ     ɛ k s p ɹ ɛ s ə ʊ
...
```

(The pronunciation of it as "eks-press-o" is proscribed, though very common: I say it myself that way.)